includes/assets/stories-widget.js

```diff
@@ -0,0 +1,121 @@
+/*
```

Comment:
I strongly believe that all javascript could live in the `assets/src` and be built using webpack. So in this case it would live in `assets/src/stories-widget`. Building using webpack, as lots of benefit, including minification and dependancy management. 

Thoughts @swissspidy

Reply:
Yes please.

No JS in `includes`.

Reply:
Moved the JS to `assets/src/widget` and added to webpack build.